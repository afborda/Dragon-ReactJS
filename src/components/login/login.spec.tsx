import React from 'react'
import { render, screen } from '@testing-library/react'

import Login from '.'

describe('<Login/>', () => {
  it('should render title', () => {
    render(<Login />)

    expect(
      screen.getByRole('heading', { name: /Abner fonseca/i })
    ).toBeInTheDocument()
  }),
    it('should render correctly', () => {
      const { container } = render(<Login />)

      expect(container.firstChild).toMatchSnapshot()
    })
})
