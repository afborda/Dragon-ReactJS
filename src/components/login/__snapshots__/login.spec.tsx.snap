// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> should render correctly 1`] = `
<div>
  <h1>
    Abner fonseca teste 
  </h1>
</div>
`
